pkgs/applications/misc/plover/default.nix

```
@@ -52,4 +69,45 @@
       makeWrapperArgs+=("''${qtWrapperArgs[@]}")
     '';
   };
+
+  plugins-manager = with python36Packages; buildPythonPackage rec {
+    pname = "plover-plugins-manager";
+    version = "0.5.16";
+
+    meta = with lib; {
+      description = "OpenSteno Plover stenography software plugin manager";
+      maintainers = with maintainers; [ tckmn ];
+      license     = licenses.gpl2;
+    };
+
+    src = fetchFromGitHub {
+      owner = "benoit-pierre";
+      repo = "plover_plugins_manager";
+      rev = "7f697da4a97cf92eeb639742a115a89887155c90";
+      sha256 = "152c02anzzvk2a5mgsnlmc1bc48lhgqf58rw2z6ir1x5carpzm91";
+    };
+
+    # apply nix sys.path wrapper to plugin manager's python call
+    postPatch = "patch -p1 <${./plugins-manager.patch}";
```

```suggestion
    postPatch = ''
      patch -p1 <${./plugins-manager.patch}
    '';
```

Why are you not using the patch in patches? It should support all options.